src/attributes.jl

```diff
@@ -337,6 +337,27 @@ An optimizer attribute for the string identifying the solver/optimizer.
 """
 struct SolverName <: AbstractOptimizerAttribute end
 
+"""
+    Silent
+
+An optimizer attribute for silencing the output of an optimizer. When `set`
+to `true`, it takes precedence over any other attribute controlling verbosity
+and requires the solver to produce no output. The default value is `false`
+which has no effect, the verbosity is controlled by other attributes.
+
+
+## Note
+
+Every optimizer should have verbosity on by default. For instance, if a solver
+has a solver-specific attribute `LogLevel` which is `0` by default, the MOI
```

Comment:
`LogLevel` is not the actual name of an attribute, so this could be a bit confusing. I would just use English, "if a solver has a solver-specific log level attribute, the MOI implementation should set it to 1 by default." ...